src/Lib.hs

```diff
@@ -296,8 +300,8 @@ performWithDefaultFlow
   -> (Bool {- verbose -}
           , NoIgnoreFlag  {- noIgnoreFlag -}
                         , SkipLocalCacheFlag {- skipLocalCache -}
-                                            , NoSkipCurrentFlag {- noSkipCurrentFlag -}
-                                                               , ConcurrentlyFlag) {- concurrentlyFlag -}
+                                            , NoSkipCurrentFlag {- skipLocalCache -}
```

Comment:
this looks like a weird rename. This is supposed to skip the current framework. Not the local cache. Unless I made a mistake that is.